src/components/footer.js

```diff
@@ -61,10 +63,20 @@ const Footer = ({
 }) => (
   <footer className={classes.footer}>
     <div className={classes.footerContent}>
+      <div className={classes.iconContainer}>
+        <a
+          className={classes.link}
+          href="https://www.linkedin.com/groups/13964590"
+          target="_blank"
```

Comment:
Let's add this to all of these links that will open in a new window:

`rel="noreferrer noopener"`

This will prevent the linked site from knowing how the page was opened. It's a helpful security measure.